app/views/missing/report.php

```diff
@@ -195,7 +195,7 @@
                                     </div>
                                     <div class="panel-body" id="panelBody4">
                                         <label for="w_phone_no">Phone number </label>
-                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" required /><br>
+                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" pattern="\d{10}" required /><br>
```

Comment:
i think this should be done in php side. From here it should be taken as string. And keep it as 8-16 digits. If its a local deployment, phone numbers can go down as 8 digits.